cf7-eideasy-attachment-signer.php

```diff
@@ -17,8 +17,6 @@
 
 function eideasy_signer_scripts()
 {
-    wp_enqueue_script('eideasy_polyfill', 'https://cdn.polyfill.io/v2/polyfill.min.js?features=Promise,fetch');
-
```

Comment:
This could be removed because:

1. It didn't work anyway as cdn.polyfill.io was hijacked and is not serving this polyfill .
2. Modern browser's support Promise and fetch natively.